test/browser/fixtures/locators/blog.test.tsx

```diff
@@ -16,6 +16,16 @@ test('renders blog posts', async () => {
   expect(firstPost.element()).toHaveTextContent(/molestiae ut ut quas/)
   expect(firstPost.getByRole('heading').element()).toHaveTextContent(/occaecati excepturi/)
 
+  expect(screen.getByRole('listitem').nth(0).element()).toHaveTextContent(/molestiae ut ut quas/)
+  var thrown
+  try {
+    screen.getByRole('listitem').nth(666)
+  } catch(err) {
+    thrown = err
```

Comment:
This doesn't actually fail, `locator` just creates a locator, it doesn't search for elements so it cannot fail

Reply:
Weird; I later verify that `thrown` is an Error with a suitable message (all synchronously) ... will investigate more closely.